lib/onc_certification_g10_test_kit/profile_selector.rb

```diff
@@ -148,6 +148,12 @@ def select_profile(resource) # rubocop:disable Metrics/CyclomaticComplexity
           return extract_profile('ObservationSocialHistory')
         end
 
+        if suite_options[:us_core_version] == 'us_core_5' &&
```

Comment:
Can we combine all the suite_options[:us_core_version] == 'us_core_5' logic together?

Reply:
We should have that, and it would affect for more than just this PR, so I'll make a follow-up ticket for that.